test/unit/connection_string.test.ts

```diff
@@ -382,9 +382,19 @@ describe('Connection String', function () {
 
   describe('validation', function () {
     it('should validate compressors options', function () {
-      expect(() => parseOptions('mongodb://localhost/?compressors=bunnies')).to.throw(
+      /*expect(() => ).to.throw(
         MongoInvalidArgumentError,
-        'bunnies is not a valid compression mechanism'
+        'bunnies is not a valid compression mechanism. Must be one of: none,snappy,zlib,zstd.'
+      ); */
```

Comment:
Let's remove the commented code, no need to keep it around

Reply:
Got it!